properties_test.go

```diff
@@ -826,6 +826,44 @@ func TestMerge(t *testing.T) {
 	assert.Equal(t, p1.GetComment("key"), "another comment")
 }
 
+func TestProperties_ToMap(t *testing.T) {
```

Comment:
Pls stick to the convention of the other tests, e.g. `TestToMap`, `input` and `p`, ....
Since `ToMap` is delivering a clone of `p.m` you can simplify the test to

```go
func TestToMap(t *testing.T) {
    input := "key=value\nabc=def"
    p := mustParse(t, input)
    m := map[string]string{"key": "value", "abc": "def"}
    assert.Equal(t, p, m)
}
```